templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <p>Logged Out</p>
+    <a href="{% url 'login' %}">Click Here to login again</a>
+{% endblock %}
```

Comment:
Add empty line

Reply:
empty lines in files login.html and logged_out.html added .